slack.md

```diff
@@ -1,147 +1,65 @@
-* **What I do.** This is shown under your picture in Slack. You can enter your official role here
-  or anything else that you think will help your coworkers understand what you do.
-* **Phone number.** Enter your phone number here. We won't use it, unless it's an emergency.
-* **Time Zone.** Be sure to specify it – it will help show Slack what the time is where you are if
-  they want to contact you.
-* **Role.** This is your official role, as per your role description.
-* **Location.** Where in the world you work from. Useful, since we're all around the globe.
-* **Team.** Which team are you a part of?
-* **Manager.** Enter the person you report to. If you are uncertain, that is usually the person you
-  have your one-to-ones with. If you can't find them, they might not be in Slack yet – make sure
-  you fill it once they are.
+If you're an engineer, make sure **your username in Slack matches the one in GitHub**. If you don't have GitHub, you can ignore this.
 
-Please fill in all of the above in your profile. Fill in the other if you'd like.
+## Channels
 
-The next step is to **upload a profile picture**. This is a **must have**, as it makes Slack
-considerably more personal. Don't just put in any picture there – use a photo where we can see your
-face and you can be identified. Bonus points if you manage to get in orange background with the
-Receipt Bank logo.
+You should already be in the following channels. Please join them if you are not.
 
-Finally, if you're a developer or working with code, make sure **your username in Slack matches the
-one in GitHub**. If you don't have GitHub, you can ignore this.
+* `#general` – Company-wide announcements and updates. What gets posted here is important, so we try to limit the volume of conversation and use threads, so that messages don’t get lost.
+* `#random` – Fun stuff. Anything that is non work related to keep us connected and remind us to be lighthearted.
+* `#hellos-goodbyes` – Say hello. Introduce yourself here.
+* `#all-hands-meeting` - Notes and recordings from all-hands meetings.
 
-That's it – you're done with your profile. Let's look at some ground rules.
+There are many channels for various topics, so it might be worth it to give a quick glance over the list and see if you're interested in any. But don't fret too much about it – if you are needed somewhere, people will invite you.
 
-## Channels
+## Creating new channels
+
+Feel free to create new channels any time you will need to. Here are some guidelines:
+
+* Separate words in the channel name with dashes `-`, not underscores `_`. Like `#baking-recipes` or `#feature-requests`.
+* Always, **always add a topic to the channel**. Note that this can only be done after the channel is created. Anybody reading it should get a good sense of what the channel is about. And remember that not everybody is in your department, so make sure that you include enough context for everybody.
```

Comment:
Channel topics can be updated at any time. 

>Note that this can only be done after the channel is created. 

<img width="1440" alt="Screenshot 2020-03-30 at 10 45 37" src="https://user-images.githubusercontent.com/879210/77887675-a992d800-7273-11ea-8274-a4319a48219a.png">
<img width="1440" alt="Screenshot 2020-03-30 at 10 45 42" src="https://user-images.githubusercontent.com/879210/77887678-abf53200-7273-11ea-9689-73114bb4be5e.png">

Reply:
I specifically removed the wrapping at 100 chars. It isn't realistic to support those guides with the wide contributing audience and keep the char-wrapping, which most people don't understand. (Like in this case, Naomi, who doesn't even have GitHub access). Besides, it's not a codebase.